README.md

```diff
@@ -559,6 +560,249 @@ The model can also be downloaded and served locally.
 
    The output from the command will consist of a series of outputs from the model before and after training.
 
+### 🧪 Evaluate the newly trained model
+
+You can use the `ilab` command to evaluate the models you are training with several benchmarks. Currently, four benchmarks are supported.
+
+| Benchmark | Measures | Full Name | Description |
+| --- | --- | --- | --- |
+| MMLU | Knowledge | Massive Multitask Language Understanding | TODO |
+| MMLUBranch | Knowledge | N/A | TODO |
+| MTBench | Skills | Multi-turn Benchmark | TODO |
+| MTBenchBranch | Skills | N/A | TODO |
+
+> [!NOTE]
+> MTBench and MTBenchBranch use [prometheus-8x7b-v2.0](https://huggingface.co/prometheus-eval/prometheus-8x7b-v2.0) as the judge model by default. While you do not **need** to use this model as your judge, it is strongly recommended. You can download it via `ilab model download`.
+
+#### Running MMLU
+
+Below is an example of running MMLU on a local model with minimal tasks:
+
+```bash
+$ export INSTRUCTLAB_EVAL_MMLU_MIN_TASKS=true   # don't set this if you want to run full MMLU 
+$ export ILAB_MODELS_DIR=/home/example-user/.local/share/instructlab/models
+$ ilab model evaluate --benchmark mmlu --model ILAB_MODELS_DIR/instructlab/granite-7b-lab
```

Comment:
Should probably add `$` to the variable name.

Reply:
Yup agreed